week-1/J-functions/exercise.js

```diff
@@ -1,7 +1,8 @@
-function halve(number) {
+function halveOfNumber(number) {
```

Comment:
Why change the function name? :) Think you were going for `halfOfNumber` here? Which would be a noun, but function names should have verbs in them, so something like `halveNumber` would work well 👍

Reply:
done